package-lock.json

```diff
@@ -54,7 +54,7 @@
 			}
 		},
 		"apps/backend": {
-			"version": "1.8.1",
+			"version": "1.10.0",
```

Comment:
backend, frontend and shared in package-lock file must have the same version that are in the main branch